helm/charts/determined/templates/NOTES.txt

```diff
@@ -37,8 +37,7 @@ format X.Y.Z (e.g., 0.13.6).
 {{ end -}}
 
 {{- if .Values.defaultScheduler }}
-      {{- $schedulerType := .Values.defaultScheduler | trim}}
-      {{- if not (or (eq $schedulerType "coscheduler") (eq $schedulerType "preemption"))}}
+      {{- if eq (.Values.defaultScheduler | trim ) "coscheduler"}}
```

Comment:
Am I reading this right - the user is only warned if they try to use "coscheduler"? Is that the intent?

Reply:
No, the helm value is only set if it's equal to 'coscheduler', otherwise it's left as blank.